components/LoginModal.vue

```diff
@@ -32,14 +32,15 @@
             label="Remember me"
             class="form__element form__checkbox"
           />
-          <span v-if="userError.login" style="color: red">
+          <span v-if="userError.login" style="color: red; display: flex; justify-content: center">
```

Comment:
Put CSS in CSS section instaed of writing it online

Reply:
Resolved